e2e/alerting_test.go

```diff
@@ -162,6 +175,11 @@ func TestCoolDown(t *testing.T) {
 	// Check that the balance enforcement was triggered using the mocked server cache.
 	posts := ts.TestSlackSvr.SlackAlerts()
 
+	sqsMessages, err := e2e.GetSNSMessages(ts.AppCfg.AlertConfig.SNSConfig.Endpoint, "alert-cooldown-test-queue")
```

Comment:
Shouldn't we test publish messages to SNS?

Reply:
This gets messages that were published by Pessimism, so this test ensures a message was published.